test/mitmproxy/proxy/protocol/test_websocket.py

```diff
@@ -272,19 +268,22 @@ def test_pong(self):
         frame = websockets.Frame.from_file(self.client.rfile)
         assert frame.header.opcode == websockets.OPCODE.PONG
         assert frame.payload == b'foobar'
+        from time import sleep
+        sleep(0.5)
+        assert self.master.has_log("Pong Received from server", "info")
```

Comment:
We could close the connection (close handshake with two close frames), after receiving a `CloseReceived` event, we should also see the log message.